themes/modern-t/modern-t.theme.bash

```diff
@@ -26,21 +28,22 @@ esac
 PS3=">> "
 
 is_vim_shell() {
-	if [ ! -z "$VIMRUNTIME" ]; then
-		echo "[${cyan}vim shell${normal}]"
+	if [ -n "$VIMRUNTIME" ]; then
+		echo "[${cyan?}vim shell${normal?}]"
 	fi
 }
 
 prompt() {
+	local last_status=$?
 	SCM_PROMPT_FORMAT='[%s][%s]'
-	if [ $? -ne 0 ]; then
+	if [ $last_status -ne 0 ]; then
```

Comment:
`IFS` can contain numbers for interactive settings. Shellcheck doesn't complain about it probably because Shellcheck doesn't see assignments to `IFS`, but `IFS` can be set externally in interactive settings.

```suggestion
	if [ "$last_status" -ne 0 ]; then
```

or

```suggestion
	if ((last_status != 0)); then
```